crates/sqlexec/src/planner/physical_plan/create_table.rs

```diff
@@ -148,9 +151,19 @@ impl CreateTableExec {
             }
         });
 
-        // Note that we're not changing out the catalog stored on the context,
-        // we're just using it here to get the new table entry easily.
-        let new_catalog = SessionCatalog::new(state);
+        // Note that we're not changing out the catalog stored on the context
+        // here. The session's catalog will get swapped out at the beginning of
+        // the next query execution.
+        //
+        // TODO: We should be returning _what_ was updated from metastore
+        // instead of needing to do this. Sean has a stash working on this.
+        let new_catalog = SessionCatalog::new(
+            state,
+            ResolveConfig {
+                default_schema_oid: 0,
+                session_schema_oid: 0,
+            },
+        );
```

Comment:
Been looking at this & i don't think i fully understand the `ResolveConfig` and what's happening here. 

AFAIK, the `ResolveConfig` is just a config object containing the global schema and session ids? 

Why are we setting them to `0` here? Is this just a temporary workaround?

Reply:
Yep that's pretty much it. The resolve config here is set to zeros since we don't actually need them, we're just creating a new session catalog to pull out the new table entry that metastore just created.

And `sqlbuiltins` is now importing quite a few crates which make it tricky to import into the `catalog` crate (cycle) and so I had to resort to the resolve config. I want to look at possibly splitting up `sqlbuiltins` to avoid that cycle, but figured it'd be a bit of a refactor.